R/targets_utils.R

```diff
@@ -138,7 +138,14 @@ make_shared_grids <- function() {
     ),
     tidyr::expand_grid(
       forecaster = "flatline_fc",
-      ahead = 1:7
+      ahead = c(1:7, 14, 21, 28)
+    ),
+    tidyr::expand_grid(
+      forecaster = "smoothed_scaled",
+      trainer = c("quantreg"),
+      ahead = c(1:7, 14, 21, 28),
+      lags = list(list(c(0, 3, 5, 7, 14), c(0),), c(0, 7, 14)),
+      pop_scaling = c(FALSE)
```

Comment:
```suggestion
      pop_scaling = FALSE
```
we don't support per-variable `pop_scaling` so avoid `c` which suggests we might, even though `c(TRUE)` and `TRUE` evaluate to the same thing.

Reply:
That was a holdover, b/c it's in an `expand_grid`, `c(FALSE,TRUE)` isn't per-variable, it's a cross product with the other lists (it's why `lags` is a list(list(c` and not just `list(c(`).